code-style/checkstyle-suppressions.xml

```diff
@@ -26,4 +27,5 @@
     <suppress files="[\\/]src[\\/]test[\\/].*" id="methodLengthMax"/>
     <suppress files="[\\/]src[\\/]test[\\/].*" id="parameterNumber"/>
     <suppress files=".*Application.java" id="hideUtilityClassConstructor"/>
+    <suppress files="ShowInternalDependencies.java" checks="HideUtilityClassConstructor"/>
```

Comment:
It's best to avoid adding to this suppressions list. You can comply with this checkstyle check by adding a private empty constructor to the class thats failing the check